src/modules/CRUD.js

```diff
@@ -0,0 +1,74 @@
+import {
+  addInput,
+  trash,
+  dots,
+  checkboxInput,
+} from './data';
+
+export function displayList() {
+  this.retrieveFromLocalStorage(this.itemInformation);
+
+  this.itemInformation.forEach((element) => {
+    document.getElementById('ulist').innerHTML += `
+    <li class="to-do-list-item flex">
+      <div class="checkbox-container">
+        <input type="checkbox" class="checkbox-input" value=${element.id}>
+        <label for="${element.listItem}" class="label"><textarea name="${element.listItem}" class="edit-list test" rows="1">${element.listItem}</textarea></label>
+      </div>
+      <i class="fa-solid fa-trash font-color target"></i>
+      <i class="font-color fa-solid fa-ellipsis-vertical display-none"></i>
+    </li>`;
+  });
+
+  for (let i = 0; i < dots.length; i += 1) {
+    trash[i].addEventListener('click', (e) => this.remove(e));
+    document.getElementsByClassName('edit-list')[i].addEventListener('keydown', (e) => this.edit(e));
+    checkboxInput[i].addEventListener('click', (e) => this.completed(e));
+  }
+
+  for (let i = 0; i < dots.length; i += 1) {
+    if (this.itemInformation[i].completed === true) {
+      checkboxInput[i].toggleAttribute('checked');
+      document.getElementsByClassName('edit-list')[i].classList.add('line-through');
+    }
+  }
```

Comment:
- [ ]  You did well coming up with this great solution, you can collect  your code inside one `loop` like this

`for (let i = 0; i < dots.length; i += 1) {
   // Your code
  }`